backend/dataline/models/connection/schema.py

```diff
@@ -112,11 +112,13 @@ def validate_dsn(value: str) -> str:
 
     # Simpler way to connect to postgres even though officially deprecated
     # This mirrors psql which is a very common way to connect to postgres
-    if value.startswith("postgres") and not value.startswith("postgresql"):
+    if value.startswith("postgres://"):
```

Comment:
This change will now fail on valid strings that specify a valid driver.

We should keep it as it was, but remove the driver perhaps.

Reply:
Can you give me an example of a valid string where this would fail?

Reply:
Just added a commit to explain what I mean, will test it out locally before merging

Reply:
Would be cool to just remove their DSN instead.
An example would be if they did: postgresql+whatever://etc.
Your changes would then fail to detect starts with 'postgres://'

<img width="691" alt="image" src="https://github.com/user-attachments/assets/6d71ab43-8255-46be-9216-ab2736b828dd">

Reply:
Yep, with driver detection and removal it makes sense